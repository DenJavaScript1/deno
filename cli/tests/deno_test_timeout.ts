function foo() {
  return new Promise((_resolve, _reject) => {
    // neither resolve nor reject is called
  });
}

Deno.test({
  name: "ever-pending promise",
  async fn() {
    await foo();
  },
  timeout: 500,
});
