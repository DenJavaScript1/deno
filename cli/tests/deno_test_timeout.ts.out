[WILDCARD]
running 1 tests
test ever-pending promise ... TIMEOUT ([WILDCARD])

failures:

ever-pending promise
TimeoutError { message: "Not finished within the 500ms timeout" }

failures:

	ever-pending promise

test result: FAILED. 0 passed; 0 failed; 1 timed out; 0 ignored; 0 measured; 0 filtered out ([WILDCARD])

